### PR TITLE
Namespaced twig syntax for views

### PR DESCRIPTION
**Pull request by dodubassman**
Same as previously done for BeelabUserPasswordBundle:

Templates can't be overriden without the namespaced twig syntax anymore.

ie : change 'BeelabUserBundle:User:index.html.twig' to '@BeelabUser/User/index.html.twig' must solve the issue.

I also did fix deprecated dependencies to `@Method` and `@Route` and a deprecated call to `AdvancedUserInterface`.

### CODE REVIEW

**Comment by garak** [comment]
Any news on this?

**Comment by garak** [comment]
Bump

File: User/UserInterface.php [hunk 1]
```diff
@@ -2,12 +2,10 @@
 
 namespace Beelab\UserBundle\User;
 
-use Symfony\Component\Security\Core\User\AdvancedUserInterface;
```
[the remark this answers is not in the document]
Comment (dodubassman) [reply User/UserInterface.php]: Hi! sorry about the delay.
It's been a while, but if i recall correclty it was a deprecated call.
Feel free to refuse it if you find this contribution useless.
regards.

[the remark this answers is not in the document]
Comment (garak) [reply User/UserInterface.php]: Yes, AdvancedUserInterface was deprecated.
Anyway, this change should be in a different PR

**Comment by garak** [comment]
This is here for a long time, a bit too much IMHO.
I'm afraid I'll have to close it, unless updated

**Pull request closed without merging**